browser/pages/confirm/confirm.html

```diff
@@ -355,6 +355,16 @@
       <span>Red Hat Container Development Kit requires Oracle VirtualBox {{checkboxModel.virtualbox.minimumVersion}} or newer 5.0.x release. Please uninstall detected Oracle VirtualBox and restart Installer to continue.</span>
     </div>
 
+    <div class="message-container error-message" ng-show="virtualization === false">
+      <span class="pficon pficon-error-circle-o"></span>
+      <span>Please enable hardware virtualization for your platform to improve performance.</span>
+    </div>
+
+    <div class="message-container error-message" ng-show="virtualization === undefined">
+      <span class="pficon pficon-error-circle-o"></span>
+      <span>Make sure hardware virtualization is enabled on your platform if supported to improve performance.</span>
```

Comment:
I would argue that even though any sort of performance is better than nothing, the error messages are sort of missing the point. Which is - virtual machines just won't run without hardware virtualization enabled. It isn't really a matter of performance. Can you get these messages to indicate that the user needs CPU virtualization enabled for the hypervisor to work properly?
(btw. sorry for neglecting this PR for so long)